zingolib/src/wallet/transactions.rs

```diff
@@ -38,3 +50,255 @@ impl TxMapAndMaybeTrees {
         self.witness_trees.as_mut().map(WitnessTrees::clear);
     }
 }
+
+pub struct ZingoAccount(AccountId, UnifiedFullViewingKey);
+
+impl Account<AccountId> for ZingoAccount {
+    fn id(&self) -> AccountId {
+        self.0
+    }
+
+    fn source(&self) -> zcash_client_backend::data_api::AccountSource {
+        unimplemented!()
+    }
+
+    fn ufvk(&self) -> Option<&UnifiedFullViewingKey> {
+        Some(&self.1)
+    }
+
+    fn uivk(&self) -> zcash_keys::keys::UnifiedIncomingViewingKey {
+        unimplemented!()
+    }
+}
+
+impl WalletRead for TxMapAndMaybeTrees {
```

Comment:
It seems like these methods are all getters.  Maybe this implementation properly belongs in `get.rs` ?

Reply:
This comment is superseded by:  #980